bitnami/kibana/8/debian-11/rootfs/opt/bitnami/scripts/kibana-env.sh

```diff
@@ -127,6 +132,16 @@ export KIBANA_FORCE_INITSCRIPTS="${KIBANA_FORCE_INITSCRIPTS:-no}"
 export SERVER_FORCE_INITSCRIPTS="$KIBANA_FORCE_INITSCRIPTS"
 export KIBANA_DISABLE_STRICT_CSP="${KIBANA_DISABLE_STRICT_CSP:-no}"
 export SERVER_DISABLE_STRICT_CSP="$KIBANA_DISABLE_STRICT_CSP"
+export KIBANA_SERVER_PUBLICBASEURL="${KIBANA_SERVER_PUBLICBASEURL:-}"
+export SERVER_PUBLICBASEURL="$KIBANA_SERVER_PUBLICBASEURL"
+export KIBANA_XPACK_SECURITY_ENCRYPTIONKEY="${KIBANA_XPACK_SECURITY_ENCRYPTIONKEY:-}"
+export XPACK_SECURITY_ENCRYPTIONKEY="$KIBANA_XPACK_SECURITY_ENCRYPTIONKEY"
+export KIBANA_XPACK_REPORTING_ENCRYPTIONKEY="${KIBANA_XPACK_REPORTING_ENCRYPTIONKEY:-}"
+export XPACK_REPORTING_ENCRYPTIONKEY="$KIBANA_XPACK_REPORTING_ENCRYPTIONKEY"
+export KIBANA_NEWSFEED_ENABLED="${KIBANA_NEWSFEED_ENABLED:-true}"
+export NEWSFEED_ENABLED="$KIBANA_NEWSFEED_ENABLED"
+export KIBANA_ELASTICSEARCH_REQUESTTIMEOUT="${KIBANA_ELASTICSEARCH_REQUESTTIMEOUT:-30000}"
+export ELASTICSEARCH_REQUESTTIMEOUT="$KIBANA_ELASTICSEARCH_REQUESTTIMEOUT"
```

Comment:
From my understanding, these variables are only for Kibana, so you only need to add this:

```
export KIBANA_SERVER_PUBLICBASEURL="${KIBANA_SERVER_PUBLICBASEURL:-}"
export KIBANA_XPACK_SECURITY_ENCRYPTIONKEY="${KIBANA_XPACK_SECURITY_ENCRYPTIONKEY:-}"
export KIBANA_XPACK_REPORTING_ENCRYPTIONKEY="${KIBANA_XPACK_REPORTING_ENCRYPTIONKEY:-}"
export KIBANA_NEWSFEED_ENABLED="${KIBANA_NEWSFEED_ENABLED:-true}"
export KIBANA_ELASTICSEARCH_REQUESTTIMEOUT="${KIBANA_ELASTICSEARCH_REQUESTTIMEOUT:-30000}"
```